maps/atlas.dmm

```diff
@@ -15289,7 +15289,7 @@
 "aYV" = (
 /obj/machinery/atmospherics/binary/valve{
 	dir = 4;
-	name = "hot loop purge valve"
+	name = "hot loop pressure release valve"
```

Comment:
I think purge valve is a better name because it's more obvious that this is the "dump all of your gas into space" valve, also it's what most people call it currently.